easybuild/easyconfigs/f/foss_mv2/foss_mv2-2016a.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'Toolchain'
+
+name = 'foss_mv2'
+version = '2016a'
```

Comment:
My idea here is to (try to) keep the same software/version of all the components of `foss` (except OpenMPI being replaced by MVAPICH2 of course). That why I believe it would make sense to use the same `<year>[ab]` version numbering.